examples/events/EiffelTestCaseTriggeredEvent/simple.json

```diff
@@ -0,0 +1,44 @@
+{
+  "meta": {
+    "type": "EiffelTestCaseTriggeredEvent",
+    "version": "1.0.0",
+    "time": 1234567890,
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee0"
+  },
+  "data": {
+    "testCase": {
+      "tracker": "My Test Management System",
+      "id": "TC-1234",
+      "version": "42",
+      "uri": "http://tm.company.com/browse/TC-1234/42"
+    },
+    "recipeId": "aaaaaaaa-bbbb-5ccc-addd-eeeeeeeeeee0",
+    "parameters": [
+      {
+        "name": "login",
+        "value": "admin"
+      }
+    ],
+    "executionType": "AUTOMATED",
+    "triggers": [
+      {
+        "type": "TIMER",
+        "description": "Nightly test timer."
+      }
+    ]
+  },
+  "links": [
+    {
+      "type": "CONTEXT",
+      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1"
+    },
+    {
+      "type": "IUT",
+      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee2"
+    },
+    {
+      "type": "TERC",
```

Comment:
My bad! No, it shouldn't be there. On a side note, schemas should check link types as well. Will have to do something about that, but first links and meta documentation should be broken down to a per-event level, too. Will create an issue for this.